packages/ptunnel-ng/0001-remove-calls-to-setuid-and-setgid.patch

```diff
@@ -0,0 +1,28 @@
+From 3973b4c77f7b21cf97d6fdb980895c2b3b557e97 Mon Sep 17 00:00:00 2001
+From: Jordan Sokolic <oofnik@gmail.com>
+Date: Wed, 7 Oct 2020 13:31:38 +0300
+Subject: [PATCH] remove calls to setuid() and setgid()
+
+---
+ src/ptunnel.c | 5 -----
+ 1 file changed, 5 deletions(-)
+
+diff --git a/src/ptunnel.c b/src/ptunnel.c
+index d70edce..6d64519 100644
+--- a/src/ptunnel.c
++++ b/src/ptunnel.c
+@@ -553,11 +553,6 @@ void* pt_proxy(void *args) {
+ #else
+ 	if (opts.uid || opts.gid)
```

Comment:
should probably remove this line too, even if the #if condition is always false it just doesn't look right

Reply:
https://github.com/lnslbrty/ptunnel-ng/blob/v1.42/src/ptunnel.c#L551-L556
i just checked and yes you definitely need to remove the preceding #if
note the lack of a closing semicolon in either condition

Reply:
tl;dr the patch should remove lines 551 to 560, it currently only removes lines 556 to 560

Reply:
also if it were up to me i would remove all references to opts.uid and opts.gid, including the -u command-line option